tensorflow/core/util/onednn_env_vars.h

```diff
@@ -20,6 +20,7 @@ limitations under the License.
 namespace tensorflow {
 bool AreWeightsFrozen();
 bool UseSystemAlloc();
+bool ThreadPoolUseCallerThread();
 }  // namespace tensorflow
```

Comment:
Nit: Would you mind adding an empty line before and after this line (one before and one after)?